README.md

```diff
@@ -8,10 +8,12 @@ This repository contains the source code for `icerpc://hello.icerpc.dev` — a p
 The following services are mapped to their default service paths and are available using the `TCP` and `QUIC`
 transports on the default port (`4062`):
 
-| Service                                       | Path                                               | Description                           | Example Clients                                                                                                                                                                                 |
-| --------------------------------------------- | -------------------------------------------------- | ------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| [Slice Greeter](./src//slice/Greeter.slice)   | `/VisitorCenter.Greeter`  | A simple service that greets visitors | [C# Secure](https://github.com/icerpc/icerpc-csharp/tree/0.3.x/examples/slice/Secure/Client) <br>[C# QUIC](https://github.com/icerpc/icerpc-csharp/tree/0.3.x/examples/slice/Quic/Client)       |
-| [Protobuf Greeter](./src/proto/greeter.proto) | `/visitor_center.Greeter` | A simple service that greets visitors | [C# Secure](https://github.com/icerpc/icerpc-csharp/tree/0.3.x/examples/protobuf/Secure/Client) <br>[C# QUIC](https://github.com/icerpc/icerpc-csharp/tree/0.3.x/examples/protobuf/Quic/Client) |
+| Service                                        | Path                        | Description                           | Example Clients                                                                                                                                                                                 |
+| ---------------------------------------------- | --------------------------- | ------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [Slice Greeter](./src//slice/Greeter.slice)    | `/VisitorCenter.Greeter`    | A simple service that greets visitors | [C# Secure](https://github.com/icerpc/icerpc-csharp/tree/0.3.x/examples/slice/Secure/Client) <br>[C# QUIC](https://github.com/icerpc/icerpc-csharp/tree/0.3.x/examples/slice/Quic/Client)       |
```

Comment:
I would give different names to the Slice and Protobuf clients, e.g.
C# Slice Secure, C# Protobuf Secure, ...